gov2/sdk/config.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//snippet-sourcedescription:[Demonstrations of the AWS SDK for Go V2 SDK features]
+//snippet-keyword:[go,config]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[10/27/2021]
+//snippet-sourceauthor:[gangwere]
```

Comment:
At a minimum, include a Purpose section, which describes the purpose of this file. It should be outside the snippet tags. This purpose is to help orient users who land here.